libcaf_core/caf/local_actor.hpp

```diff
@@ -340,12 +340,12 @@ class CAF_CORE_EXPORT local_actor : public monitorable_actor {
   /// Serializes the state of this actor to `sink`. This function is
   /// only called if this actor has set the `is_serializable` flag.
   /// The default implementation throws a `std::logic_error`.
-  virtual error save_state(serializer& sink, unsigned int version);
+  virtual error save_state(serializer& sink, unsigned int version) const;
 
   /// Deserializes the state of this actor from `source`. This function is
   /// only called if this actor has set the `is_serializable` flag.
   /// The default implementation throws a `std::logic_error`.
-  virtual error load_state(deserializer& source, unsigned int version);
+  virtual error load_state(deserializer& source, unsigned int version) const;
```

Comment:
```suggestion
  virtual error load_state(deserializer& source, unsigned int version);
```

A const load function makes no sense, since it's purpose is to override the state of an actor.

Reply:
Thank you I oversee this as a quick refactor. Thank you I reverted back this changes.